ferminet/hamiltonian.py

```diff
@@ -173,25 +174,25 @@ def potential_nuclear_nuclear(charges: Array, atoms: Array) -> jnp.ndarray:
     charges: Shape (natoms). Nuclear charges of the atoms.
     atoms: Shape (natoms, ndim). Positions of the atoms.
   """
-  r_aa = jnp.linalg.norm(atoms[None, ...] - atoms[:, None], axis=-1)
+  natoms = atoms.shape[0]
+  r_aa = jnp.sqrt(jnp.sum((atoms[None, ...] - atoms[:, None] + 
+      jnp.eye(natoms)[..., None])**2, axis=-1))
```

Comment:
Can you also undo this change so we just jnp.linalg.norm still here (just less surprising to the reader) please?